### ISSUE TITLE
Crash every time the app is opened

### ISSUE THREAD

**Issue by s1m-e0n** [nextcloud/android#10017]
Every time I installed the Nextcloud app at some point it would crash with the error message below. The crash would then occur everytime the app is opened, clearing the cache or app data wouldn't help. I have reinstalled the app at least 10 times already, so it is very frequent. This occurs on the dev build as well.

### Steps to reproduce
There is no consistent pattern, it will randomly happen. I think though, that it always happened on a loading screen, so when the app was waiting on something, though I'm not entirely sure. There are no settings that I changed, the last crash occured pretty much right after I logged in.

### Expected behaviour
- The app does not crash. The app won't crash every time it's reopened.

### Actual behaviour
- The app would unpredictably crash while using it, even if the thing you were doing worked previously. Then, after reopening the app it crashes again with the same error massage. This does not affect background sync however, the background sync will continue working.

### Environment data
Android version: 8.0.0

Device model: Huawei RNE-L21

Stock or customized system: Huawei EMUI 8.0.0

Nextcloud app version: 3.19.1 (downloaded from F-Droid)

Nextcloud server version: 23.0.2

Reverse proxy: Nginx

#### Nextcloud client log

```
************ CAUSE OF ERROR ************

java.lang.IllegalStateException: Couldn't read row 0, col 3 from CursorWindow.  Make sure the Cursor is initialized correctly before accessing data from it.
	at android.database.CursorWindow.nativeGetString(Native Method)
	at android.database.CursorWindow.getString(CursorWindow.java:440)
	at android.database.AbstractWindowedCursor.getString(AbstractWindowedCursor.java:51)
	at android.database.CursorWrapper.getString(CursorWrapper.java:137)
	at com.owncloud.android.datamodel.FileDataStorageManager.createFileInstance(FileDataStorageManager.java:988)
	at com.owncloud.android.datamodel.FileDataStorageManager.getFileByPath(FileDataStorageManager.java:124)
	at com.owncloud.android.datamodel.FileDataStorageManager.getFileByEncryptedRemotePath(FileDataStorageManager.java:111)
	at com.owncloud.android.datamodel.FileDataStorageManager.getFileByPath(FileDataStorageManager.java:107)
	at com.owncloud.android.datamodel.FileDataStorageManager.saveFile(FileDataStorageManager.java:255)
	at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:252)
	at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:363)
	at java.lang.Thread.run(Thread.java:784)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30190190
Build flavor: generic

************ DEVICE INFORMATION ************
Brand: HUAWEI
Device: HWRNE
Model: RNE-L21
Id: HUAWEIRNE-L21
Product: RNE-L21

************ FIRMWARE ************
SDK: 26
Release: 8.0.0
Incremental: 354(C432)

```

**Comment by AlvaroBrey** [comment]
Duplicate of #9948
